src/css/var/getStyles.js

```diff
@@ -1,7 +1,7 @@
 define( function() {
 	return function( elem ) {
 
-		// Support: IE<=11 only, Firefox<=30+ (#15098, #14150)
+		// Support: IE <=11 only, Firefox <=30+ (#15098, #14150)
```

Comment:
I'll need to verify first if it's still broken in current Firefox. I mostly tested IE/Edge in these commits.